File: practica/C++/1.1.cpp
#include <iostream>
#include <cmath>

#include <windows.h>
#include <conio.h>

using namespace std;

int main()
{
    SetConsoleCP(1251);
    SetConsoleOutputCP(1251);

    cout << "Введіть x: ";
    double x;
    cin >> x;

    double y = log2( fabs( (pow(x, 2) - 4) / (pow(x, 2) - 1) ) ); 
    cout << "Розрахунок формули рівний y: " << y << "\n";

    getch();

    return 0;
}

File: practica/python/1.1.py
from msvcrt import getch
from math import pow, log2, fabs

x = float(input("Введіть х: "))

#розрахунок формули
y = log2( fabs( (pow(x, 2) - 4) / (pow(x, 2) - 1) ) )
print("Розрахунок формули рівний y: ", y, sep = "")

getch()